src/binded.js

```diff
@@ -0,0 +1,6 @@
+const methodReducer = obj => (state, methodName) =>
+  Object.assign({}, state, { [methodName]: obj[methodName].bind(obj) });
```

Comment:
I’ve told you should check if `obj[methodName]` is a `Function`. Any own enumerable key of `obj` could be of any type and then you’ll get `ReferenceError` saying something has no method `bind`.